src/components/Alert/Alert.tsx

```diff
@@ -69,7 +69,13 @@ export const Alert = ({
   );
 
   return (
-    <div className={classes} data-testid='notification' {...properties}>
+    <div
+      className={classes}
+      data-testid='notification'
+      aria-live='polite' // default accessibility setting, can be overridden by the properties spread
```

Comment:
Heyo! We use alerts everywhere, from classic alerts telling users something terrible has happened, to a lot of alerts that just live on the page as just containers for information and shouldn't be announced early to a screenreader: legal/beta language, upload success when revisiting a page, success messages on sign and submit, etc. 

I'd be worried about making everything default to this `aria-live` attribute, and would rather have it set intentionally. Also, if you wanted to interrupt a screenreader with `assertive` for an alert, you wouldn't want to set `aria-live` to `assertive` here in this prop, but rather set the `role` to `alert`, which acts like if `aria-live` is `assertive` and `aria-atomic` is `true` and gives you more semantic html to boot. To do that I'd have to overwrite these defaults and then set the `role=alert`.

I could imagine us having these aria labels / roles being set by the `status` (which could still be overwritten), so that `error` set `role=alert`, `warning` set `aria-live='polite'` etc. But I'd probably wait to do that post-mvp after we talk with Natalia a bit about how Alerts are used across the app, to make things more consistent / potentially revive a separate "notification" component we had talked about earlier, talk with the other team using these components about how they use it, etc. This feels like a good discussion post-mvp about pushing people towards more accessible apps through sensible defaults.

Let me know what you think. For now, I'd probably just stick with adding your own aria properties in sbl-frontend.